Google.PowerShell.IntegrationTests/BigQuery/GcbqCmdlets.ps1

```diff
@@ -4,5 +4,9 @@ Install-GcloudCmdlets
 # Define variables that will be used in testing
 $nonExistProject = "project-no-exist"
 $accessErrProject = "asdf"
+$oneDaySec = 86400
```

Comment:
This is fine as-is, but consider the following to be more readable. (1) Don't just include a constant, but the values used to derive it. (2) Use partial values to make it more readable.

````
$oneDaySec = 60 * 60 * 24;
$oneDayMs = $oneDaySec * 1000
$threeDaySec = 3 * $oneDaySec
$threeDayMs = 3 * $oneDayMs
````